crates/oxc_cli/src/command/lint.rs

```diff
@@ -147,6 +146,16 @@ pub enum OutputFormat {
     Json,
 }
 
+impl FromStr for OutputFormat {
+    type Err = String;
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        Ok(match s {
+            "json" => Self::Json,
+            _ => Self::Default,
```

Comment:
I'd do something like
```rust
Ok(match s {
    "json" => Self::Json,
    "default" => Self::Default,
    _ => return Err(format!("{s:?} is not a known format")),
})
```

This would make it so `--format json` and `--format default` are accepted but `--format pancake` is rejected.

Reply:
Thank you! I totally missed it. Landed in c250b28.